api/v1alpha1/olsconfig_types.go

```diff
@@ -103,13 +102,12 @@ type ConversationCacheSpec struct {
 	Type CacheType `json:"type,omitempty"`
 	// +optional
 	Redis RedisSpec `json:"redis,omitempty"`
-	// +optional
-	Memory MemorySpec `json:"memory,omitempty"`
 }
 
 // RedisSpec defines the desired state of Redis.
 type RedisSpec struct {
 	// Redis host
+	// +kubebuilder:default=lightspeed-redis-server.openshift-lightspeed.svc
 	Host string `json:"host,omitempty"`
```

Comment:
Redis host and port option do we need to have as configurable in CR ? We will not use anyway , we compute the service address as default right ?

Reply:
Yes, I thought of doing that cleanup in subsequent PRs. But have updated now.